lib/logstash/outputs/scalyr.rb

```diff
@@ -341,6 +345,16 @@ def multi_receive(events)
           sleep_interval *= 2
           retry if @running
         end
+
+        if !@exc_data.nil?
+          message = "Retry successful after error."
```

Comment:
Would also be good to include how many retry attempts we used and overall delay / sleep time.